example.gpt

```diff
@@ -0,0 +1,3 @@
+Tools: github.com/gptscript-ai/answers-from-the-internet
+
+Which countries were the top 5 gold medal winners in the 1996 summer olympics?
```

Comment:
Not pressing for me, but maybe worth doing something that couldn't be pulled out of the LLMs memory. We could try something like "What are the current top 10 songs in the U.S right now?".

Reply:
I tried out the olympics question and it worked pretty well. It found and used the right sources. I like it because it has a precise and correct answer to check for.